src/main/java/com/code_intelligence/jazzer/runtime/TraceCmpHooks.java

```diff
@@ -69,6 +73,14 @@ final public class TraceCmpHooks {
     TraceDataFlowNativeCallbacks.traceCmpLong(thisObject, (long) arguments[0], hookId);
   }
 
+  @MethodHook(type = HookType.BEFORE, targetClassName = "clojure.lang.Numbers", targetMethod = "isZero")
+  @MethodHook(type = HookType.BEFORE, targetClassName = "clojure.lang.Numbers", targetMethod = "isPos")
+  @MethodHook(type = HookType.BEFORE, targetClassName = "clojure.lang.Numbers", targetMethod = "isNeg")
+  public static void staticLongCompareZero(
```

Comment:
You can keep these hooks here, they are very simple and similar to existing hooks.